packages/node/test/unit/fixture/native/node_modules/hello_world/package.json

```diff
@@ -5,19 +5,16 @@
   "main": "hello.js",
   "private": true,
   "dependencies": {
-    "node-gyp-build": "4.8.0"
+    "node-gyp-build": "4.8.4"
   },
   "scripts": {
     "install": "node-gyp-build",
     "build": "npm run build:darwin && npm run build:win32 && npm run build:linux",
-    "prebuildify": "prebuildify --napi --strip --target 16.20.0",
+    "prebuildify": "npx prebuildify --napi --strip --target 18.13.0",
```

Comment:
(must) prevent it from installing latest and running :(

Reply:
wait, what exactly do you want here?

Reply:
@naugtur To be clear this is _not_ run in CI and is only run by a developer in a development environment when the fixtures must be updated, which is unlikely to happen often.